### ISSUE TITLE
[Feature Request] Merge all Slimbook apps into one app

### ISSUE THREAD

**Issue by Tarek-Hasan** [Slimbook-Team/slimbookbattery#122]
Please, merge battery, face, gestures, RGB etc. into one app. This will reduce complication of using different apps for few simple tasks. Developing a single app will also reduce maintenance burden and reduce wastage of developers time.

You can introduce "Slimbook Control Center" (or whatever name you wish) similar to [Tuxedo Control Center](https://github.com/tuxedocomputers/tuxedo-control-center) by Tuxedo Computers.

**Comment by piranna** [comment]
I disagree, the apps serve different and independent purposses, at most I would find it useful a meta-package that have all the other apps as dependencies, so all of them can be installed at once.
